src/toolbars/examples/toolbar.js

```diff
@@ -50,7 +50,7 @@
   <file name="index.html">
     <div ng-controller="ViewCtrl" class="row example-container">
       <div class="col-md-12">
-        <pf-toolbar id="exampleToolbar" config="toolbarConfig">
+        <pf-toolbar id="exampleToolbar" config="toolbarConfig" on-toggle-colvis="toggleColvis(column)">
          <actions>
```

Comment:
Up until now pfToolbar has been independent of any 'view' (list, card).  It seems wrong to add a table-view specific attribute here.  Especially,  since (and I agree with @jeff-phillips-18) it's name is very cryptic.   Could we rename it to make it clearer?  How about 'toggleTableColsVisibility'.  I know 'colvis' is the name the plug-in uses, but it has been depreciated, and soon will be forgotten.